### ISSUE TITLE
Feature Request: Hide Window Title Bar

### ISSUE THREAD

**Issue by SoilnRock** [milkdrop2077/MilkDrop3#14]
In windowed mode it would be awesome to hide the window title bar :-)

PS: If this is not the place for feature requests, please say so!

**Comment by milkdrop2077** [comment]
It's done! you have the beta version right? press F7 twice

**Comment by SoilnRock** [comment]
Great! Unfortunately windows moving/resizing doesn't work in "no borders" mode.

**Comment by Tatsuya79** [comment]
There could also be a windowed fullscreen option / fullscreen not always on top mode.

Then you can play an emulator on top of it, the sound from retroarch is working here for me:
![Screenshot_170](https://github.com/milkdrop2077/MilkDrop3/assets/6266038/98bc5e3c-c011-4b1a-adcc-f35ccfd166a8)

thanks for that great program btw 😃

**Comment by milkdrop2077** [comment]
interesting idea :)

**Comment by milkdrop2077** [comment]
Hey I made an option that should solve your problem:
I made windowed fullscreen option / fullscreen not always on top mode in MilkDrop 3.21
Start MilkDropPanel, and set AlwaysOnTop to "Always OFF".
Restart MilkDrop.
Now when MilkDrop will be in fullscreen, it won't be always on the top, meaning you could bring up any other window on  top of MilkDrop!
Try it

**Comment by milkdrop2077** [comment]
+ now moving/resizing works in "no borders" mode!

**Comment by OfficialIncubo** [comment]
> Hey I made an option that should solve your problem:
> I made windowed fullscreen option / fullscreen not always on top mode in MilkDrop 3.21
> Start MilkDropPanel, and set AlwaysOnTop to "Always OFF".
> Restart MilkDrop.
> Now when MilkDrop will be in fullscreen, it won't be always on the top, meaning you could bring up any other window on  top of MilkDrop!
> Try it

Yeah. Every BeatDrop/Milkdrop3 has this too. If you are in fullscreen, press F7 twice, so it will enter the windowed fullscreen mode.

**Comment by Tatsuya79** [comment]
It doesn't seem to be working here.
I selected "fullscreen" and always on top = "always OFF" in MilkDropPanel,
but when I start milkdrop I can not bring any window on top.

If I try with F7 while it's in fullscreen, when I reach always on top OFF it brings back the window decoration.
If then I hit alt+enter, it goes into a smaller window.
Then alt+enter again; it goes fullscreen with nothing that can be displayed on top again.

It's strange as I managed to do it at start once with a sequence of window/fullscreen/option tests...

**Comment by Tatsuya79** [comment]
OK, I found how it worked once.

Select "fullscreen" and always on top = "always OFF" in MilkDropPanel.
Start Milkdrop (it's in fullscreen but you can't bring anything on top of it).
Do alt+enter twice to go windowed then fullscreen again.

Now it's working you can bring other windows on top of it.

If Milkdrop starts with always on top enabled, I can never bring anything on top of it,
even if I change it with F7 and switch the window/fullcreen state several times.

**Comment by SoilnRock** [comment]
> * now moving/resizing works in "no borders" mode!

Awesome!!!

**Comment by milkdrop2077** [comment]
@Tatsuya79 
Press Alt and then Tab a few time to switch windows, select the window you want to bring to the front with Tab, you should have MilkDrop3 in the back and anything you want in the front. Maybe hide the taskbar in the windows settings too

**Comment by Tatsuya79** [comment]
I know about alt+tab! 😄 
I have to do alt+enter twice or it doesn't work.

**Comment by milkdrop2077** [comment]
Hey @Tatsuya79 @SoilnRock can you try this updated version?
https://mega.nz/file/BIcXHBJI#UFEkjH_2EXGtEprJ37miC6nzh0L-7gUkmJ6wpORn3zc

 @Tatsuya79 I've changed quite a bite of code to suits your need,
now just check the Disable AlwaysOnTop in MilkDropPanel, and restart milkdrop, when you're in fullscreen, alt+tab should bring anything you want on the top.

Also please check if the options / about / donate are launching properly on your system.
And at last try to change the font size in the MilkDropPanel to see if it works!
Thanks

**Comment by Tatsuya79** [comment]
It's still the same: starting in fullscreen with disable Alwaysontop, I can't bring anything on top.

Starting in window with disable Alwaysontop, then doing alt+enter, it works, I can bring stuff on front of milkdrop.

The donate window is working.

Changing the font size worked once. Then it stays on those values forever, it doesn't save any additional change.

**Comment by milkdrop2077** [comment]
@Tatsuya79 ok thanks I found the problem and I've now fixed the starting in fullscreen with disable Alwaysontop.

>>Changing the font size worked once: 
If you change the number and restart the MilkDropPanel, can you see your new numbers in the MilkDropPanel? 
Can you check if the FontSizeDefault= FontSizeInfo= are updated in the Milk3.ini file? 
Did you restart MilkDrop3 to check the font size? 
thanks

**Comment by milkdrop2077** [comment]
I've published MilkDrop 3.22 that should fix all the issues.
You can now also launch MilkDrop directly with your emulator!

**Comment by Tatsuya79** [comment]
Awesome it's working!
Great idea that launch option! 😄

**Comment by OfficialIncubo** [comment]
Is there anyway a C++ code of how I can make the borderless mode resizeable?
Tell me on Twitter DMs or on Discord: Incubo_#4768.
